pyrsa/rdm/rdms.py

```diff
@@ -240,3 +252,79 @@ def subsample(self, by, value):
                     rdm_descriptors=rdm_descriptors,
                     pattern_descriptors=pattern_descriptors)
         return rdms
+
+    def append(self, rdm):
+        """ appends an rdm to the object
+        The rdm should have the same shape and type as this object.
+        Its pattern_descriptor and descriptor are ignored
+
+        Args:
+            rdm(pyrsa.rdm.RDMs): the rdm to append
+
+        Returns:
+
+        """
+        assert isinstance(rdm, RDMs), 'appended rdm should be an RDMs'
+        assert rdm.n_cond == self.n_cond, 'appended rdm had wrong shape'
+        assert rdm.dissimilarity_measure == self.dissimilarity_measure, \
+            'appended rdm had wrong dissimilarity measure'
+        self.dissimilarities = np.concatenate((
+            self.dissimilarities, rdm.dissimilarities), axis=0)
+        self.rdm_descriptors = append_descriptor(self.rdm_descriptors,
+                                                 rdm.rdm_descriptors)
+        self.n_rdm = self.n_rdm + rdm.n_rdm
+    def save(self, filename):
+        """ saves the RDMs object into files
+
+        This function generates two files:
+            [filename].npy : a numpy array file to save the dissimilarities
+            [filename]_desc.pkl : a pickled file of the descriptors
+
+        Args:
+            filename(String): path to file to save to
+
+        """
+        np.save(filename + '.npy', self.dissimilarities)
+        pickle.dump([self.descriptors,
```

Comment:
If you go down this route, instead of a list, you could use a dictionary. This way you don't have to maintain the same order in the future and you'll save yourself headaches

Reply:
I like the dictionary idea. Will implement this soon